### PR TITLE
💬 Update reset password template text

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: src/utils/url.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ export function wrapUtm(url: string) {
   return `${url}?utm_medium=email&utm_source=email`;
 }
 
+export function wrapCrispUtm(url: string) {
+  return `${url}&utm_medium=email&utm_source=email`;
+}
+
```
Comment (nwingt) [inline src/utils/url.ts]: think should combine this function into `wrapUtm()` and `<CrispLink/>` into `<Link/>` to provide option as second param/props to choose `?` or `&`

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/utils/url.ts (hunk 1 above)
Comment (shellyYG) [reply src/utils/url.ts]: gotcha! Changed in next commit

**Comment by shellyYG** [comment]
Ah forgot to do this one: '<CrispLink>' into '<Link>'
Am doing this ask now

File: src/utils/url.ts [hunk 2]
```diff
@@ -1,11 +1,7 @@
 import { ASSETS_ROOT } from '../constants';
 
-export function wrapUtm(url: string) {
-  return `${url}?utm_medium=email&utm_source=email`;
-}
-
-export function wrapCrispUtm(url: string) {
-  return `${url}&utm_medium=email&utm_source=email`;
+export function wrapUtm(url: string, prefix: string) {
```
Comment (nwingt) [inline src/utils/url.ts]: maybe default is '?', so that you can skip inputting it?

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/components/link.tsx [hunk 3]
```diff
@@ -9,16 +9,34 @@ export interface LinkProps
...
+  const {
+    children,
+    href,
+    style,
+    isWrapUtm = false,
+    isCrispLink = false,
+    ...restProps
+  } = props;
+  let hrefToPass = href;
+  if (href && isWrapUtm && isCrispLink) {
```
Comment (nwingt) [inline src/components/link.tsx]: Perhaps adding another prop for controlling the prefix, e.g. wrapUtmPrefix,
```javascript
wrapUtm(href, wrapUtmPrefix)
```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/components/link.tsx (hunk 3 above)
Comment (nwingt) [reply src/components/link.tsx]: @shellyYG

File: src/utils/url.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { ASSETS_ROOT } from '../constants';
 
-export function wrapUtm(url: string) {
-  return `${url}?utm_medium=email&utm_source=email`;
+export function wrapUtm(url?: string, wrapUtmPrefix: string = '?') {
+  return `${url}${wrapUtmPrefix}utm_medium=email&utm_source=email`;
```
Comment (nwingt) [inline src/utils/url.ts]: Can keep naming `prefix` for simplicity￼ 😉
```suggestion
export function wrapUtm(url?: string, prefix: string = '?') {
  return `${url}${prefix}utm_medium=email&utm_source=email`;
```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/utils/url.ts (hunk 4 above)
Comment (shellyYG) [reply src/utils/url.ts]: gotcha! changed :D

**Review by nwingt — APPROVED** [review]

**Pull request merged**